app/views/layouts/_header.html.erb

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-default navbar-fixed-top navbar-inverse">
```

Comment:
I removed the `fixed` nature of this navbar because it was problematic for the env banner. It was not a useful quality and was also causing us some z-index issues.